### DISCUSSION TITLE
Welcome to bodymiscale Discussions!

### DISCUSSION THREAD

**Discussion by dckiller51** [dckiller51/bodymiscale#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by azrael783** [comment]
Hey,

I'm Daniel, 37 years old and I life in Germany. I really like this integration due to the fact that it automatically fetches the data from the scale and I don't have to open up the MI app to have my values stored. Thx for creating this awesome integration :-)

**Reply by dckiller51** [reply]
Thanks for your feedback. Also watch here
https://community.home-assistant.io/t/custom-component-bodymiscale-for-xiaomi-miscale-1-2-esphome-or-ble-monitor/253542

**Comment by ghost** [comment]
Hi, I happen to have worked on a project that is hugely related to yours: https://github.com/foormea/esp32_mi_scale_2
Happy to work together with you on making things even better!

**Reply by dckiller51** [reply]
Nice work, my initial project is the integration into the Esphome environment.

[Xiaomi Miscale](https://esphome.io/components/sensor/xiaomi_miscale.html) 

[Xiaomi Miscale2](https://esphome.io/components/sensor/xiaomi_miscale2.html)
